data/snippets.conf

```diff
@@ -58,6 +66,11 @@ try=try%block%\ncatch (%cursor%)%block_cursor%
 if=if (%cursor%)%block_cursor%
 else=else%block_cursor%
 for=for (int i = 0; i < %cursor%; i++)%brace_open%\n%brace_close%
+syso=System.out.println("%cursor%");
+Syso=System.out.println("%cursor%");
+main=package %cursor%;\n\npublic class Main {\n\tpublic static void main(String[] args) {\n\t\t%cursor%\n%brace_close1%%brace_close%
```

Comment:
In this case, maybe change the brace position on others because for Java with Oracle convention we put brace on the same line than class/method or control instruction. I don't have propose this change only to don't change actual snippet.